test/runner.html

```diff
@@ -81,51 +82,39 @@
       // 'unit/multi-style.html'
     ];
 
-    // http://eddmann.com/posts/cartesian-product-in-javascript/
-    function flatten(arr) { return [].concat.apply([], arr) }
-    function product(sets) {
-      return sets.reduce(function(acc, set) {
-        return flatten(acc.map(function(x) {
-           return set.map(function(y) { return x.concat(y); });
-        }));
-      }, [[]]);
-    }
     function combinations(suites, flags) {
-      return product(flags).reduce(function(list, f) {
-        f = f.filter(function(i) { return i; }).join('&');
-        return list.concat(suites.map(function(s) { return s + (f ? '?' + f : '') }))
-      }, []);
+      return flags.map((f) => {
+        return f ? suites.map(s => `${s}?${f}`) : suites;
+      }).reduce((arr, s) => arr.concat(s), []);
     }
 
-    function addUrlOption(previous, next) {
-      previous = previous || '';
-      next = next || '';
+    function addUrlOption(previous = '', next = '') {
       return previous + (previous ? '&' : '') + next;
     }
 
     // test shadowdom/custom elements polyfills together
     // preferring both if possible.
-    var matrix = [''];
+    let flags = [''];
     if (window.customElements) {
-      matrix.push('wc-ce=true');
+      // flags.push('wc-ce=true');
```

Comment:
Let's remove or explain why we're not testing this.

Reply:
Done